Mathlib/Analysis/NormedSpace/Alternating/Basic.lean

```diff
@@ -0,0 +1,609 @@
+
+end ContinuousLinearMap
+
+theorem LinearIsometry.norm_compContinuousAlternatingMap (g : F →ₗᵢ[𝕜] G) (f : E [⋀^ι]→L[𝕜] F) :
+    ‖g.toContinuousLinearMap.compContinuousAlternatingMap f‖ = ‖f‖ :=
+  g.norm_compContinuousMultilinearMap f.1
+
+open ContinuousAlternatingMap
+
+section
+
+theorem ContinuousAlternatingMap.norm_compContinuousLinearMap_le (f : F [⋀^ι]→L[𝕜] G)
+    (g : E →L[𝕜] F) : ‖f.compContinuousLinearMap g‖ ≤ ‖f‖ * (‖g‖ ^ Fintype.card ι) :=
+  (f.1.norm_compContinuousLinearMap_le _).trans_eq <| by simp
+
+/-- Composition of a continuous alternating map and a continuous linear map
+as a bundled continuous linear map. -/
+def ContinuousAlternatingMap.compContinuousLinearMapCLM (f : E →L[𝕜] F) :
+    (F [⋀^ι]→L[𝕜] G) →L[𝕜] (E [⋀^ι]→L[𝕜] G) :=
+  LinearMap.mkContinuous
+    (ContinuousAlternatingMap.compContinuousLinearMapₗ f) (‖f‖ ^ Fintype.card ι) fun g ↦
+      (g.norm_compContinuousLinearMap_le f).trans_eq (mul_comm _ _)
+
+/-- Given a continuous linear isomorphism between the domains,
+generate a continuous linear isomorphism between the spaces of continuous alternating maps. -/
+@[simps apply]
+def ContinuousLinearEquiv.continuousAlternatingMapCongrLeft (f : E ≃L[𝕜] F) :
+    E [⋀^ι]→L[𝕜] G ≃L[𝕜] (F [⋀^ι]→L[𝕜] G) where
+  __ := f.continuousAlternatingMapCongrLeftEquiv
+  __ := ContinuousAlternatingMap.compContinuousLinearMapCLM (f.symm : F →L[𝕜] E)
+  toFun g := g.compContinuousLinearMap (f.symm : F →L[𝕜] E)
+  continuous_invFun :=
+    (ContinuousAlternatingMap.compContinuousLinearMapCLM (f : E →L[𝕜] F)).cont
+  continuous_toFun :=
+    (ContinuousAlternatingMap.compContinuousLinearMapCLM (f.symm : F →L[𝕜] E)).cont
+
+variable
+  {E' : Type*} [NormedAddCommGroup E'] [NormedSpace 𝕜 E']
+  {F' : Type*} [NormedAddCommGroup F'] [NormedSpace 𝕜 F']
+
+/-- Continuous linear equivalences between the domains and the codomains
+generate a continuous linear equivalence between the spaces of continuous alternating maps. -/
+@[simps! apply]
+def ContinuousLinearEquiv.continuousAlternatingMapCongr (e : E ≃L[𝕜] E') (e' : F ≃L[𝕜] F') :
+    (E [⋀^ι]→L[𝕜] F) ≃L[𝕜] (E' [⋀^ι]→L[𝕜] F') :=
+  e.continuousAlternatingMapCongrLeft.trans <| e'.continuousAlternatingMapCongrRight
+
+end
+
+open ContinuousAlternatingMap
+
+namespace AlternatingMap
+
+/-- Given a map `f : F →ₗ[𝕜] E [⋀^ι]→ₗ[𝕜] G` and an estimate
+`H : ∀ x m, ‖f x m‖ ≤ C * ‖x‖ * ∏ i, ‖m i‖`, construct a continuous linear
+map from `F` to `E [⋀^ι]→L[𝕜] G`.
+
+In order to lift, e.g., a map `f : (E [⋀^ι]→ₗ[𝕜] F) →ₗ[𝕜] E' [⋀^ι]→ₗ[𝕜] G`
+to a map `(E [⋀^ι]→L[𝕜] F) →L[𝕜] E' [⋀^ι]→L[𝕜] G`,
+one can apply this construction to `f.comp ContinuousAlternatingMap.toAlternatingMapLinear`
+which is a linear map from `E [⋀^ι]→L[𝕜] F` to `E' [⋀^ι]→ₗ[𝕜] G`. -/
+def mkContinuousLinear (f : F →ₗ[𝕜] E [⋀^ι]→ₗ[𝕜] G) (C : ℝ)
+    (H : ∀ x m, ‖f x m‖ ≤ C * ‖x‖ * ∏ i, ‖m i‖) : F →L[𝕜] E [⋀^ι]→L[𝕜] G :=
+  LinearMap.mkContinuous
+    { toFun := fun x => (f x).mkContinuous (C * ‖x‖) <| H x
```

Comment:
```suggestion
    { toFun x := (f x).mkContinuous (C * ‖x‖) <| H x
```
(and throughout - this has become legal syntax since you first wrote this)

Reply:
Done.